modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,22 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option allows you to split the current container block element when the enter key is pressed, and specify which blocks should be split.
```

Comment:
This change still isn't correct sorry @MitchCTiny. It's not something like allows in addition to, it's one or the other.
```suggestion
This option allows specifying how the current container block element is split when the enter key is pressed. It can either, always split the block, split for specific block elements or not split at all.
```

I think this should cover it better, but not too sure on the wording so I'd like to get @brian-forte's input here.